### PR TITLE
Fixed Ace Editor unknown option warning

### PR DESCRIPTION
**Pull request by airarrazaval**
Fix for issue https://github.com/formio/formio.js/issues/4164

### CODE REVIEW

File: src/components/_classes/component/Component.js [hunk 1]
```diff
@@ -2249,7 +2249,7 @@ export default class Component extends Element {
       .then((editor) => {
         editor = editor.edit(element);
         editor.removeAllListeners('change');
-        editor.setOptions(settings);
+        editor.setOptions(_.omit(settings, 'isUseWorkerDisabled'));
```
Comment (travist) [inline src/components/_classes/component/Component.js:2252]: I would rather use ```_.pick``` so we won't have to keep introducing new "omits" as we add other settings elsewhere.

**Review by travist — CHANGES_REQUESTED** [review]

**Comment by airarrazaval** [comment]
@travist that's a pretty big list:

https://github.com/ajaxorg/ace/wiki/Configuring-Ace

Some options are missing but are defined in the [types declaration file](https://github.com/securingsincity/react-ace/blob/main/types.d.ts#L41-L98) and in their respectove controller files (see below).

I will set the default options in the `wysiwygDefault.ace` object and then do something like 

```js
editor.setOptions(_.pick(settings, Object.keys(this.wysiwygDefault.ace)));
```

I took the default values from the following ace options definitions:

* [Editor Options](https://github.com/ajaxorg/ace/blob/35ddb50f353f14ba56872b5921d95d81e1ad7f08/lib/ace/editor.js#L2861)
* [Renderer Options](https://github.com/ajaxorg/ace/blob/35ddb50f353f14ba56872b5921d95d81e1ad7f08/lib/ace/virtual_renderer.js#L1742)
* [MouseHandler Options](https://github.com/ajaxorg/ace/blob/145328b96083c6b7c306d3dcf841948f73486919/lib/ace/mouse/mouse_handler.js#L214)
* [Session Options](https://github.com/ajaxorg/ace/blob/145328b96083c6b7c306d3dcf841948f73486919/lib/ace/edit_session.js#L2460)

**Comment by travist** [comment]
Oh wow... I didn't realize how big this list was. Let me discuss this internally and see which approach we like better. Thanks for your efforts on this.

**Comment by travist** [comment]
So don't hate me, but I think I like the other way better now. This does not really save us any more maintenance and arguably would add more (where we would need to add a new property as they are added to ace). I would recommend reverting it back to the way it was and we will commit it.

**Comment by heather-formio** [comment]
PR is out of date. Closing, please re-open new PR if still needed. Thank you for your contribution.

**Pull request closed without merging**